src/services/redux/features/cookieStatus/cookieStatusSlice.ts

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+import { ICookieStatus } from "../../../../types/redux/StoreTypes";
+
+const initialState: ICookieStatus = {
+	status: "pending",
+};
+
+const cookieStatus = createSlice({
+	name: "cookieStatus",
+	initialState,
+	reducers: {
+		setCookieStatus(state, action) {
```

Comment:
state: ICookieStatus, action: PayloadAction<CookieStatusList>
better add types for reducer